features/tile-switcher-animation.json

```diff
@@ -0,0 +1,7 @@
+{
+    "_meta": {
+        "description": "A feature flag that determines whether the tile switcher info panel animation is enabled.",
```

Comment:
This seems like a sub feature, does it need to be a parent one?

Reply:
Actually it's a parent. The name of the json though should probably be tab-switcher-animation.json rather than `tile`

Reply:
sorted

Reply:
My question is why isn’t it a sub feature?

Reply:
I'll add to my guidance doc, ideally we should be raising draft config ahead of time and testing it in the client too.